docs/troubleshooting.md

```diff
@@ -0,0 +1,212 @@
+# Collector Troubleshooting
+
+This document describes common error conditions that Collector may encounter, and
+provides steps that can be taken to diagnose the cause, fix potential issues and
+return Collector to a working state.
+
+If the troubleshooting steps below do not solve the issue, please contact ACS
+support.
+
+## First Steps
+
+### Logs
+
+The first thing to look at are the logs from any failing Collectors. The troubleshooting
+guides below will offer examples of log messages you can expect to see for a number of
+error conditions, so having access to your own logs is invaluable.
+
+Depending on the environment, and accesses, you can get these in a number of ways.
+
+#### Kubectl
+
+Kubernetes is the easiest way to retrieve the logs, provided you have access to
+do so. 
+
+```sh
+$ kubectl get pods -n stackrox | grep collector
+collector-vclg5                      1/2     CrashLoopBackOff   2 (25s ago)   2m41s
+
+$ kubectl logs -n stackrox collector-vclg5 collector
+```
+
+#### OC
+
+Since the `oc` tool maintains compatibility with kubectl, the required commands are
+much the same.
+
+```sh
+$ oc get pods -n stackrox | grep collector
+collector-vclg5                      1/2     CrashLoopBackOff   2 (25s ago)   2m41s
+
+$ oc logs -n stackrox collector-vclg5 collector
+```
+
+#### Diagnostic Bundle
+
+An alternative method is to download a diagnostic bundle from the ACS UI, by
+following the instructions [here](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/3.72/html-single/configuring/index#generate-diagnostic-bundle)
```

Comment:
Perhaps change 3.72 to latest.

Reply:
I'm not sure this is possible but I've asked around and hopefully there's a way of doing it